ompi/mca/coll/han/coll_han_component.c

```diff
@@ -392,6 +398,31 @@ static int han_register(void)
                                               "low level module for scatterv, 0 basic",
                                               OPAL_INFO_LVL_9, &cs->han_scatterv_low_module,
                                               &cs->han_op_module_name.scatterv.han_op_low_module_name);
+    cs->han_alltoall_low_module = 0;
+    (void) mca_coll_han_query_module_from_mca(c, "alltoall_lower_module",
+                                              "low level module for alltoall, 0 tuned, 1 sm ",
+                                              OPAL_INFO_LVL_9, &cs->han_alltoall_low_module,
+                                              &cs->han_op_module_name.alltoall.han_op_low_module_name);
+    cs->han_alltoall_pstages = 0;
+    (void) mca_base_component_var_register(c, "alltoall_pstages",
+                                              "Parallel Stages for alltoall.  Higher numbers require more memory, "
+                                              "and performs more communication in parallel.  0 chooses pstages based on message size.",
+                                              MCA_BASE_VAR_TYPE_INT32_T, NULL, 0, 0,
+                                              OPAL_INFO_LVL_9, MCA_BASE_VAR_SCOPE_READONLY,
+                                              &cs->han_alltoall_pstages);
+    cs->han_alltoall_subdivfactor = 0;
+    (void) mca_base_component_var_register(c, "alltoall_subdivfactor",
+                                              "Sub-divide Factor for alltoall.  1 puts pressure on SHM, 4 puts pressure on NIC. "
+                                              "Larger numbers require more rounds, but each round is faster.",
+                                              MCA_BASE_VAR_TYPE_INT32_T, NULL, 0, 0,
+                                              OPAL_INFO_LVL_9, MCA_BASE_VAR_SCOPE_READONLY,
+                                              &cs->han_alltoall_subdivfactor);
+    cs->han_alltoall_algorithm = 0;
+    (void) mca_base_component_var_register(c, "alltoall_algorithm",
+                                              "Algorithm selection. 0=auto-based-on-size, 1=allgather-based, 2=xpmem-based.",
```

Comment:
I removed the allgather-based algorithm, but a forgot to update the documentation.  Note to myself here!

Reply:
Fixed.  This whole configuration block shouldn't have existed anyways, since I should have used `coll_han_use_alltoall_algorithm` instead.